tests/security_features_test.go

```diff
@@ -279,9 +279,9 @@ var _ = Describe("[Serial][sig-compute]SecurityFeatures", Serial, decorators.Sig
 		})
 	})
 	Context("Disabling the custom SELinux policy", func() {
-		var policyRemoved = false
+		var policyRemovedByTest = false
 		AfterEach(func() {
-			if policyRemoved {
+			if policyRemovedByTest {
```

Comment:
I think we could remove this compliantly but I am ok with following up here

Reply:
The policy is actually considered installed by virt-handler, and won't get auto-reinstalled.

Reply:
Maybe we should reset the internal boolean when we see the feature gate appear.